src/ao3/works.py

```diff
@@ -263,6 +266,73 @@ def bookmarks(self):
         # bookmarked this, but for now just return the number.
         return int(self._lookup_stat('bookmarks').contents[0])
 
+    @property
+    def bookmark_users(self):
+        api_url = ('https://archiveofourown.org/works/%s/bookmarks?page=%%d'
+        % self.id)
+
+        usernames = []
+        pseuds = []
+
+        num_users = 0
+        for page_no in itertools.count(start=1):
+            print("Finding page: \t" + str(page_no) + " of bookmarks. \t" + str(num_users) + " usernames found.")
+
+            req = self.sess.get(api_url % page_no)
```

Comment:
Rather than using string substitutions, you can use the params argument:

```suggestion
            req = self.sess.get(api_url, params={"page": page_no})
```